leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -717,10 +772,11 @@ var ElementInspector = Backbone.View.extend({
      * constraints from previously stored.
      */
     addConstraint: function(event) {
-
+        
```

Comment:
This function needs to be renamed because it is not adding a constraint, it is adding a function segment. Same goes for the class name that calls this function